make/tools.mk

```diff
@@ -54,7 +57,7 @@ arm_sdk_install: arm_sdk_download $(SDK_INSTALL_MARKER)
 $(SDK_INSTALL_MARKER):
 ifneq ($(OSFAMILY), windows)
         # binary only release so just extract it
-	$(V1) tar -C $(TOOLS_DIR) -xjf "$(DL_DIR)/$(ARM_SDK_FILE)"
+	$(V1) tar -C $(TOOLS_DIR) -xf "$(DL_DIR)/$(ARM_SDK_FILE)"
```

Comment:
The file is still .bz2 / .zip. maybe `-a, --auto-compress` ?

Reply:
For Windows it's zip, otherwise it has changed:

https://developer.arm.com/downloads/-/arm-gnu-toolchain-downloads